monkey/infection_monkey/exploit/web_rce.py

```diff
@@ -154,21 +155,23 @@ def exploit(self, url, command):
         """
         raise NotImplementedError()
 
-    def get_open_service_ports(self, port_list, names):
+    @staticmethod
+    def get_open_service_ports(victim_host: VictimHost, port_list, names):  # noqa: F821
```

Comment:
If this is being modified anyway, might as well add type hints for other arguments and the return value as well.